tests/unittest/cloud_governance/policy/helpers/test_policy_operations.py

```diff
@@ -17,20 +17,17 @@ def test_verify_and_delete_resource_not_stopped():
     :rtype:
     """
     environment_variables.environment_variables_dict['DAYS_TO_TAKE_ACTION'] = 3
-    environment_variables.environment_variables_dict['policy'] = 'instance_run'
+    environment_variables.environment_variables_dict['policy'] = 'ip_unattached'
```

Comment:
are u sure 'ip_unattached' is ok ?

Reply:
This is for testing the methods, so I added one of the policy to make it test pass.